website/docs/user-guides/uninstall-litmus.md

```diff
@@ -27,36 +27,42 @@ To disconnect the [ChaosAgent](../getting-started/resources.md#chaosagents) conn
    If the ChaosAgent is not reachable it would remove only the entry from the database of the ChaosCenter
    :::
 
-### Removing the respective components individually
+### Remove the CRs
 
-To remove the respective components of the ChaosAgents you need to manually delete the created resources of that ChaosAgent.
+To remove the CRs Litmus uses, use the following command:
+
+- To remove individual CRs
+
+```bash
+kubectl delete chaosexperiments <CHAOSEXPERIMENTS_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete chaosresults <CHAOSRESULTS_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete workflows <WORKFLOW_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete cronworkflows <CRONWORKFLOW_NAMEs> -n <AGENT_NAMESPACE>
```

Comment:
There are a few more CDRs like event tracker...etc. Here is the link for all CRDs -https://github.com/litmuschaos/litmus/blob/master/litmus-portal/litmus-portal-crds.yml

Reply:
Have added that at the bottom with the title `Remove the Litmus CRDs`